docs/tutorial/queries.rst

```diff
@@ -152,127 +164,163 @@ there in the ``actors`` by using a :ref:`computable
-    .........     num_actors := count(Movie.actors)
-    ......... };
+    edgedb> SELECT Movie {
+    .......     title,
+    .......     num_actors := count(Movie.actors)
+    ....... };
     {
-        Object { title: 'Blade Runner 2049', num_actors: 3 },
-        Object { title: 'Dune', num_actors: 0 }
+      default::Movie {title: 'Blade Runner 2049', num_actors: 3},
+      default::Movie {title: 'Dune', num_actors: 0},
     }
 
 Let's add some more information about "Dune". For example, we can add
 some of the actors, like Jason Momoa, Zendaya and Oscar Isaac:
 
 .. code-block:: edgeql-repl
 
-    tutorial> INSERT Person {
-    .........     first_name := 'Jason',
-    .........     last_name := 'Momoa'
-    ......... };
-    Object { id: <uuid>'618d4cd6-54db-11e9-8c54-67c38dbbba18' },
-    tutorial> INSERT Person {
-    .........     first_name := 'Oscar',
-    .........     last_name := 'Isaac'
-    ......... };
-    Object { id: <uuid>'618d5a64-54db-11e9-8c54-9393cfcd9598' },
+    edgedb> INSERT Person {
+    .......     first_name := 'Jason',
+    .......     last_name := 'Momoa'
+    ....... };
+    default::Person {id: 618d4cd6-54db-11e9-8c54-67c38dbbba18}
+    edgedb> INSERT Person {
+    .......     first_name := 'Oscar',
+    .......     last_name := 'Isaac'
+    ....... };
+    default::Person {id: 618d5a64-54db-11e9-8c54-9393cfcd9598}
 
 Unfortunately, adding Zendaya is not possible with the current schema
 since both ``first_name`` and ``last_name`` are required. So let's
-:ref:`alter <ref_eql_ddl_object_types_alter>` our schema to make
-``first_name`` optional. We'll use :ref:`DDL <ref_eql_ddl>` here
-for brevity as it lets us modify the schema without a new migration:
+migrate our schema to make ``first_name`` optional.
 
-.. code-block:: edgeql-repl
+First, we'll update the ``movie_schema/schema.esdl``:
 
-    tutorial> ALTER TYPE Person {
-    .........     ALTER PROPERTY first_name {
-    .........         SET OPTIONAL;
-    .........     }
-    ......... };
-    ALTER
+.. code-block:: sdl
+
+    module default {
+        type Person {
+            property first_name -> str;
+            required property last_name -> str;
+        }
+        type Movie {
+            required property title -> str;
+            # the year of release
+            property year -> int64;
+            required link director -> Person;
+            multi link actors -> Person;
+        }
+    };
 
-Now we can add Zendaya:
+Second, let's create a new migration to this new schema state:
+
+.. code-block:: bash
+
+    $ edgedb create-migration --schema-dir movie_schema
+    did you make property 'first_name' of object type
+    'default::Person' optional? [y,n,l,c,b,s,q,?]
+    y
+    Created movie_schema/migrations/00002.edgeql, id:
+    m15dogciav42crguwkslmjoucq5ieg25hcgk7ewxfmcmzf7othksda
+
+Third and final step in this sequence is to apply the migration:
+
+.. code-block:: bash
+
+    $ edgedb migrate --schema-dir movie_schema
```

Comment:
`--schema-dir` -- I don't think we need this in the tutorial. We should use the default location.

Reply:
You're right that we can just use the default for this tutorial.

I wonder if it's still worth mentioning that the directory can be specified via this option, just to reassure a newcomer that they still can pick some sensible location for their schema in the project.

Reply:
> I wonder if it's still worth mentioning that the directory can be specified via this option, just to reassure a newcomer that they still can pick some sensible location for their schema in the project.

I don't think we should. We can't cover all side questions that readers might have -- the tutorial would become hard to read. So let's keep it as simple as possible and to the point.